### DISCUSSION TITLE
Apply to derive bathimetry

### DISCUSSION THREAD

**Discussion by susilosarimun** [opengeos/lidar#13] [Q&A] [answered]
Dear Qiusheng Wu,

My Name Susilo, I want to ask about this software.
Have you try use this software to processing Lidar batimetry data to derive bathimetry in the coastal area?

Thank you very much

Susilo

**Comment by giswqs** [comment, marked as answer]
Yes, you can use the package to process bathymetric lidar dem. See https://lidar.gishub.org/usage
